### PR TITLE
Liaison fb

### PR DESCRIPTION
**Pull request by Marie-Garde**

### CODE REVIEW

**Review by Thoms-C — DISMISSED** [review]
Seems ok !!

**Comment by LoicBrassart** [comment]
Solve conflicts before assigning me plz

**Review by Gauthier-Lhomme — DISMISSED** [review]

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_DB_HOST=
```
Comment (LoicBrassart) [inline .env.sample]: In a frontend app, you **never** mention databases ! :scream:

File: .env.sample [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+BACKEND_PORT=
```
Comment (LoicBrassart) [inline .env.sample]: This, on the other end, seems legit. But the variable name *must* begin with `REACT_APP_` to be recognized :shrug:

File: package.json [hunk 3]
```diff
@@ -6,6 +6,9 @@
     "@testing-library/jest-dom": "^5.11.6",
     "@testing-library/react": "^11.2.2",
     "@testing-library/user-event": "^12.5.0",
+    "axios": "^0.21.1",
+    "axios-hooks": "^2.3.0",
```
Comment (LoicBrassart) [inline package.json]: What's this package ?

File: src/App.js [hunk 4]
```diff
@@ -28,6 +28,7 @@ export default function App() {
         <Route path="/action" component={Action} />
         <Route path="/inscription" component={Inscription} />
         <Route path="/action" component={ActionPage} />
+        <Route path="/BDDFavart" component={Link} />
```
Comment (LoicBrassart) [inline src/App.js]: `<Link>` being a named component from `react-router`, I **really** recommend that you change this name (and once again, why mention the database at all ?)

File: src/components/DisplayBDD.jsx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+          <Company>Company</Company>
+        </Text>
+      </FlexBox1>
+      <FlexBox2>
+        <div>
+          {loading && <p>it's loading</p>}
+          {!loading && (
+            <>
+              {clients.map((client, i) => (
+                <Card>
```
Comment (LoicBrassart) [inline src/components/DisplayBDD.jsx]: IIRC, the `key={id}` prop should be on this line, not on it's infants

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some fixes to do !

**Review by Gauthier-Lhomme — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
NmE

**Pull request merged**